.github/workflows/verify_integrations_map_added.yml

```diff
@@ -0,0 +1,34 @@
+name: Verify integrations map correctly added
+
+on:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  bump_package_versions:
+    runs-on: windows-latest
+
+    steps:
+      - name: Support longpaths
+        run: git config --system core.longpaths true
+
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: '6.0.100'
+
+      - name: "Regenerating package versions"
+        run: .\tracer\build.ps1 GeneratePackageVersions
+
+      - name: Report error
+        run: |
+          echo "## :warning: Error generating package versions" >> $GITHUB_STEP_SUMMARY
```

Comment:
Just to make sure, I'm not missing something. This will be run all the time even when there are no errors, right?

Reply:
oops 🤦‍♂️